identity/src/test/java/com/android/identity/mdoc/mso/StaticAuthDataTest.java

```diff
@@ -14,20 +14,13 @@
  * limitations under the License.
  */
 
```

Comment:
Since we're leaving the existing `Utility.java` functions alone, I would leave the tests in `legacy/StaticAuthDataTest.java` alone too and add new tests as `mso/StaticAuthDataTest.java`